examples/shell/cmd_btp.cpp

```diff
@@ -0,0 +1,181 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <core/CHIPCore.h>
+
+#if CONFIG_NETWORK_LAYER_BLE
+
+#if CONFIG_DEVICE_LAYER
+#include <platform/CHIPDeviceLayer.h>
+#endif
+
+#include <shell/shell.h>
```

Comment:
On nRF Connect build this includes shell.h from Zephyr instead of CHIP. A simple fix is to use `#include <lib/shell/shell.h>` though there's an issue to address that differently.

Reply:
Thanks a lot!